README.rst

```diff
@@ -14,6 +14,10 @@ bookstore and lending library in Paris.
 
 Python 3.5 / Django 1.11 / Node 10.5.0 / MariaDB (MySQL) 5.5+ w/ timezone info
 
+.. image:: https://github.com/Princeton-CDH/mep-django/workflows/ci/badge.svg
+    :target: https://github.com/Princeton-CDH/mep-django/actions?query=workflow%3Aci
+    :alt: CI status
```

Comment:
Lets give the workflow and the badge alt more meaningful names. Travis badge says "build passing". I don't care about duplicating that exactly, but not everyone knows what CI stands for.

Reply:
can you say more about what audience you think the workflow name should be intelligible to? ime badges are usually used to court open source developers who are thinking about contributing to a project, which is a demographic that is pretty likely to recognize "ci" (although it doesn't really match how we contribute to CDH projects). i didn't choose "build" because - other than the webpack files - the workflow isn't producing any artifacts, and we're not working with compiled code. happy to go with a different name if it makes sense.